pkg/metallb/mlbtypes/bpgpeer.go

```diff
@@ -0,0 +1,94 @@
+package mlbtypes
```

Comment:
I thought everything was going to be put into the types pkg?

Reply:
We need to import types from this package and later we will have the same types under argocd pacakge. So in order to avoid name duplication and added mlb prefix.